scandium-core/src/main/java/org/eclipse/californium/scandium/DatagramFilterExtended.java

```diff
@@ -0,0 +1,40 @@
+/*******************************************************************************
+ * Copyright (c) 2022 Bosch.IO GmbH and others.
```

Comment:
Are you an employee of Bosch.IO? 

If not, please edit this according [license_header_template.txt](https://github.com/eclipse-californium/californium/blob/main/license_header_template.txt). if this is your own work not belonging to an employer, then just use your name. 

There is also a pending PR #2097 . If you like, you may chose also that. I plan to replace all header when start to work in the 4.0.